ui/apps/platform/src/Containers/Dashboard/PatternFly/SummaryCounts.tsx

```diff
@@ -0,0 +1,76 @@
+import React from 'react';
+import Raven from 'raven-js';
+import pluralize from 'pluralize';
+import { useQuery } from '@apollo/client';
+import { Spinner, Split, Stack } from '@patternfly/react-core';
+
+import { SummaryCountsResponse, SUMMARY_COUNTS } from 'queries/summaryCounts';
+
+type SummaryTileProps = {
+    label: string;
+    value: number;
+    isLoading: boolean;
+    isError: boolean;
+};
+
+/*
+  TODO
+  We have a `TileContent` component that was used in the old version of this
+  widget at ./apps/platform/src/Components/TileContent/TileContent.tsx
+
+  Is this a pattern we want to retain in new PatternFly pages?
+
+  PF has a similar looking `Tile` component, but it has different semantics from what
+  we are doing here: https://www.patternfly.org/v4/components/tile
+*/
+function SummaryTile({ label, value, isLoading, isError }: SummaryTileProps) {
+    let content;
+    if (isLoading) {
+        content = <Spinner size="lg" aria-label={`Total ${label} count`} />;
+    } else if (isError) {
+        content = <p title={`Failed to load ${label} counts`}>- -</p>;
```

Comment:
The `- -` text here hasn't been run by UXD yet, but I wanted to something to cover the error case in the PR.

Reply:
What are pro and con to lift up the loading and error states from `SummaryTile` to `SummaryCounts` because all of the tiles depend on one request?
* `if (loading) { return …; }`
* `if (error) { return …; }`
* `return …;`

Reply:
That was how I initially approached this, but I moved the loading states to the `SummaryTile` so that I could display loading and error states per tile. Moving these up to `SummaryCounts` would make the code a bit simpler, but would require an all-or-nothing widget to display loading and error states for the entirety of the tiles. Another question for UX - I don't think any of the mocks cover the loading/error states.

Reply:
In this case, the loading and error state is all-or-nothing from summary counts request, true?

Reply:
Yes that is true, the loading/error state will always be identical, at the same time, for each individual tile.

Reply:
Hmm good question. Gut feeling is that if this is the case, the individual fields would need to be a union type with `number` and something else, but I'll see if I can find out for sure.

Reply:
Ok so it looks like in-theory we could get partial errors with this query, but the implementation sends a single error back if any part fails out:

```graphql
query q {
	clusterCount
	nodeCount
	violationCount
	deploymentCount
	imageCount(query: "Last Updated:>a")
	secretCount
}
```
returns a response of 
```json
{
	"errors": [
		{
			"message": "converting to bleve query: Invalid time query. Must be of the format (01/02/2006 or 1d)",
			"path": [
				"imageCount"
			]
		}
	],
	"data": null
}
```